kitchen.recipes-config.yml

```diff
@@ -291,3 +291,39 @@ suites:
         - network_interfaces_configuration_script_created
         - network_interfaces_configured
         - multiple_network_interfaces_can_ping_gateway
+  - name: custom_slurm_settings_file_via_s3
```

Comment:
Can we put a small comment here briefly describing the scope of the test?

Reply:
Done